### DISCUSSION TITLE
Failed to fetch resource 404

### DISCUSSION THREAD

**Discussion by PatLeosrisook** [community/community#122086] [Programming Help]
### Select Topic Area

Question

### Body

Hello, 
I've been trying the figure out what went wrong with my website (Build with react) for awhile now. I tried many solutions I found but it still doesn't work (This includes, correcting file path, checking if DNS is correctly configured, moving index.html to root directory, etc...) 

here is the error I got : 
![Capture-2024-05-03-111042 AM](https://github.com/community/community/assets/49037327/1ae4cb0b-1f8e-4e3a-8846-a2de29b3488c)

I have no idea what's ' main.css' and 'main.js' are?

Here's my file structure
![File structure](https://github.com/community/community/assets/49037327/8f0d46cf-60d4-4130-9a38-98e0f1f912e2)

Here's what I meant when I say correcting file path (I change it from having './' or '../' in front. It a solution I found on [This stackoverflow question](https://stackoverflow.com/questions/68497926/how-can-i-fix-failing-to-load-resource-status-404-github-pages)) 
![Code example ](https://github.com/community/community/assets/49037327/726e828b-4e08-46d7-9595-76eb9490513c)

and here is the path to manifest.json and letter_logo.ico. Are they incorrect ?
![CleanShot 2024-05-03 at 11  18 09@2x](https://github.com/community/community/assets/49037327/9d82c600-0068-4c9b-9351-fd64c91c0660)

And my CNAME in my DNS is also correctly pointed at my github url. 

I don't know what I'm missing.. If there any part of code you'd like to see let me know!

Thank you in advance

**Comment by LiteBrite82** [comment]
Hi @PatLeosrisook 

Thanks for being a part of the community!!!

I see some issues in your code; Im gonna send this over to the Programming Help category for maximum exposure.

**Reply by PatLeosrisook** [reply]
Thank you!

Could you point out the issue? It's been almost a week now and I still can't figure out what it is 🥲

**Reply by LiteBrite82** [reply]
Put index.html in the `public` folder

**Reply by PatLeosrisook** [reply]
Still not working :( 

Could the problem be the syntax error in manifest.json file? I can't find where that file is on my computer.
